orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/WaitForUpInstancesTaskSpec.groovy

```diff
@@ -325,15 +323,14 @@ class WaitForUpInstancesTaskSpec extends Specification {
     )
 
     where:
-    result || useSource | healthy | configured | snapshot | asg | description
```

Comment:
some of these tests fail - i started cleaning it up to understand what they actually test - `useSourceCapacity` isn't used anyway, so removing it